docs/api-overview.mdx

```diff
@@ -41,12 +41,16 @@ The wallet url for the Production environment is `https://wallet.verified.inc`.
 While developing your integration you can use a dummy user account if you would like.
 
 ```json test
-email: richard@piedpiper.net
-phone: +10123456789
+email: richard@piedpiper.net || richard.hendricks@verified.inc
+phone: +10123456789 || +550123456789
 ```
 
 By using these test identifiers you can authenticate to the wallet using a static OTP value of `111111`, allowing you to test the full flow of the integration more easily. We recommend using a proper test email or phone to recieve actual OTP authentication messages during the final stages of development.
 
+:::caution
+Credentials issued to the test user will be deleted every day at 12am EST (UTC-5).
```

Comment:
```suggestion
Credentials issued to the test user will be deleted every day at 07:00 UTC.
```

Reply:
The cron only nukes the test users creds right? Not manually issued test creds by customeres?

Reply:
Yeah, just the test user creds